pkg/volume/iscsi/iscsi.go

```diff
@@ -105,10 +105,10 @@ func (plugin *iscsiPlugin) newBuilderInternal(spec *volume.Spec, podUID types.UI
 			iqn:     iscsi.IQN,
 			lun:     lun,
 			manager: manager,
-			mounter: &mount.SafeFormatAndMount{mounter, exec.New()},
 			plugin:  plugin},
 		fsType:   iscsi.FSType,
 		readOnly: readOnly,
+		mounter:  &mount.SafeFormatAndMount{mounter, exec.New()},
```

Comment:
I know I've seen `exec.New()` elsewhere in the code, so a future PR, perhaps, but it's probably better for plugins to obtain an exec interface from the plugin host.